test/e2e/engine/template.go

```diff
@@ -102,10 +102,12 @@ func Build(cfg *config.Config, masterSubnetID string, agentSubnetID string, isVM
 		}
 	}
 
-	if config.PublicSSHKey != "" {
-		cs.ContainerService.Properties.LinuxProfile.SSH.PublicKeys[0].KeyData = config.PublicSSHKey
-		if cs.ContainerService.Properties.OrchestratorProfile.KubernetesConfig != nil && cs.ContainerService.Properties.OrchestratorProfile.KubernetesConfig.PrivateCluster != nil && cs.ContainerService.Properties.OrchestratorProfile.KubernetesConfig.PrivateCluster.JumpboxProfile != nil {
-			cs.ContainerService.Properties.OrchestratorProfile.KubernetesConfig.PrivateCluster.JumpboxProfile.PublicKey = config.PublicSSHKey
+	if cs.ContainerService.Properties.LinuxProfile != nil {
```

Comment:
can we assign cs.ContainerService.Properties to a variable?
```suggestion
	props := cs.ContainerService.Properties
```